src/lib/y2storage/proposal/autoinst_nfs_planner.rb

```diff
@@ -187,25 +206,29 @@ def mandatory_values(format)
       # Name of the server from a NFS share
       #
       # @param share [String] e.g., "192.168.56.1:/root_fs"
-      # @return [String]
+      # @return [String] e.g., "192.168.56.1"
       def server(share)
         server_and_path(share).first || ""
```

Comment:
does "" makes sense here? I think that nfs without server is not possible.

Reply:
Well, the idea is to create the NFS with the value given in the profile. If no server was given, then we will create it without a server name. libstorage-ng will complain later if the NFS share cannot be mounted.